RELEASE.md

```diff
@@ -150,7 +160,8 @@ Once the workflow job is done, ensure all publication sources have the new artif
 ### 5. Download artifacts
 
 Once the full publish-ci.yml worflow is done, the release artifacts can be downloaded from the GitHub Artifacts (located in the "Summary" tab of the action).
-These are currently manual steps taken from the browser.
+Similarly, release artifacts can be downloaded from the docs-ci.yml workflow.
```

Comment:
We'll need to update the docs-ci.yml workflow to run on push to `release/v*` branches.

Reply:
We can add "release/v*" to the push trigger, but what about Upload Artifact and Deploy?  

Should it be:
- Upload Artifact on main and release
- Deploy only on release

Reply:
It should be "Upload on push to main or release/v*; deploy on push main" until we have a proper directory structure in place (future PR).